### PR TITLE
Feature/5298: Move uplinkset type and LIG/LI checks to init (no need to redo them)

### PR DESCRIPTION
**Pull request by stenioaraujo**

### CODE REVIEW

File: networking_oneview/ml2/drivers/oneview/mech_oneview.py [hunk 1]
```diff
@@ -38,6 +38,12 @@ def __init__(self):
             CONF.oneview.uplinkset_mappings)
         self.flat_net_mappings = common.load_conf_option_to_dict(
             CONF.oneview.flat_net_mappings)
+
+        common.check_uplinkset_types_constraint(
```
Comment (hugonicodemos) [inline networking_oneview/ml2/drivers/oneview/mech_oneview.py]: move those checks to initialize

File: tox.ini [hunk 2]
```diff
@@ -30,9 +30,8 @@ deps = pylint
 # C0103: Invalid (method, variable, constant) name
 # E0401: Unable to import
 # R0914: Too many local variables: need refactor
-# R0913: Too many arguments: remove this in feature/5298
 commands =
-  pylint -f parseable -d C0111,C0103,E0401,R0914,R0913 networking_oneview --ignore=tests,db
+  pylint -f parseable -d C0111,C0103,E0401,R0914 networking_oneview --ignore=tests,db
```
Comment (hugonicodemos) [inline tox.ini]: +2

File: networking_oneview/ml2/drivers/oneview/mech_oneview.py (hunk 1 above)
Comment (hugonicodemos) [inline networking_oneview/ml2/drivers/oneview/mech_oneview.py]: move those checks to initialize.

**Review by hugonicodemos — CHANGES_REQUESTED** [review]

File: networking_oneview/ml2/drivers/oneview/mech_oneview.py (hunk 1 above)
Comment (ricardoas) [reply networking_oneview/ml2/drivers/oneview/mech_oneview.py]: agreed

**Review by ricardoas — CHANGES_REQUESTED** [review]

**Review by hugonicodemos — APPROVED** [review]

**Review by SauloAislan — APPROVED** [review]

**Review by ricardoas — APPROVED** [review]

**Review by GustavoDinizMonteiro — APPROVED** [review]

**Review by ricardoas — APPROVED** [review]

**Pull request merged**